static/css/perdiem.css

```diff
@@ -62,16 +62,16 @@ body {
 
 .nav {
     list-style: none;
-    padding: 8px 16px;
     margin: 0;
-    background-color: #ddd;
+    background-color: #transparent;
```

Comment:
I think here you mean:
`background-color: transparent;` (without the `#`)
This might work in Chrome but some other browsers might not display correctly.